trailblazer/cli/core.py

```diff
@@ -69,31 +69,38 @@ def log_cmd(context, sampleinfo, sacct, quiet, config):
 
 
 @base.command()
-@click.option('-c', '--mip-config', type=click.Path(exists=True), help='MIP config')
+@click.option('-c', '--mip-config', 'pipeline_config', type=click.Path(exists=True),
+              help='Pipeline config')
 @click.option('-e', '--email', help='email for logging user')
 @click.option('-p', '--priority', type=click.Choice(['low', 'normal', 'high']), default='normal')
 @click.option('-d', '--dryrun', is_flag=True, help='only generate SBATCH scripts')
-@click.option('--command', is_flag=True, help='only show the MIP command')
-@click.option('-sw', '--start-with', help='start the pipeline beginning with program,\
-                                           see format for program in mip.pl')
+@click.option('--pipeline', help='sub-pipeline to run')
+@click.option('--script', help='binary path')
+@click.option('--command', is_flag=True, help='only show the pipeline command')
```

Comment:
I now this ain't new. `command` is so meaningless. Can we rename it to `show-command` or something more descriptive?

Reply:
same here, I have as a goal in this PR not to break the application :)

Reply:
Alright!